src/decorators/withResources/Resource.js

```diff
@@ -9,11 +9,13 @@ import {
   HTTP_VERB_PATCH,
   HTTP_VERB_POST,
   HTTP_VERB_DELETE,
+  ALL_HTTP_VERBS,
```

Comment:
Nitpick: Personally I think this name is a bit misleading; I thought it was going to be an array of all the supported methods; maybe `HTTP_VERB_WILDCARD` or `HTTP_VERB_ALL_WILDCARD` or something?